### DISCUSSION TITLE
Page icon support

### DISCUSSION THREAD

**Discussion by scimone** [ramnes/notion-sdk-py#77] [Q&A] [answered]
Hi,
The newest Notion Version "2021-08-16" supports updating the icons of pages.
However, when I try to do it via this sdk like this, nothing happens:
`data = {'page_id':  page_id, 'icon': {'type': 'emoji', 'emoji': "🎉"}`
`response = client.pages.update(**data)`

Is it just not implemented yet or am I doing something wrong?

Thank you.

**Comment by ramnes** [comment, marked as answer]
I did not release 2021-08-16 support yet, but it's on the master branch. Can you try with the latest commit?

**Reply by scimone** [reply]
Ok makes sense ^^ 
Is it easy to build the python package from the latest commit? Otherwise I will just wait until the next release. Thank you for your valuable work!

**Reply by ramnes** [reply]
`pip install git+https://github.com/ramnes/notion-sdk-py.git@4dc60c491b51d152fdb93caa0efd00d91f30adb9` should work. :)
